sim/cartpole.py

```diff
@@ -8,15 +8,7 @@
 # pyright: strict
 
 import math
-import os
 import random
-import sys
-
-from bonsai3 import ServiceConfig, SimulatorInterface, SimulatorSession
```

Comment:
Subsequent to this PR, cartpole-py was changed to use bonsai-common (#6). That differentiates this repo from the [cartpole sample in the microsoft-bonsai-api repo](https://github.com/microsoft/microsoft-bonsai-api/tree/main/Python/samples/cartpole).

If we agree that bonsai-common is appropriate for this repo, then the API changes in this PR would no longer be needed.

Reply:
Agreed. I turned this PR into a draft because there was confusion as to which sample was going to be microsoft-bonsai-api v. bonsai-common. I agree this one is bonsai-common. I wrote some initial documentation using this branch for quickstart python integration for the cookbook. I'm happy to abandon this PR.